rllib/algorithms/ars/ars_tf_policy.py

```diff
@@ -85,10 +85,10 @@ def __init__(self, obs_space, action_space, config):
             for _, variable in self.variables.variables.items()
         )
 
-    def compute_actions(self, observation, add_noise=False, update=True, **kwargs):
+    def compute_actions(self, obs_batch, add_noise=False, update=True, **kwargs):
```

Comment:
I feel like it's not worth it to change this parameter name.
it's a little clearer for sure, but that will break all user scripts that currently call these APIs.
unless you intentionally want to break things, to force migration, we should probably keep things backward compatible?

Reply:
My issue is that ARS and ES are the only places where we use `observation` instead of `obs_batch` as the name of the arg. But you are wright - let's move this elsewhere.

Reply:
Thanks!